packages/flutter_android_lifecycle/pubspec.yaml

```diff
@@ -0,0 +1,60 @@
+name: flutter_android_lifecycle
+description: A new flutter plugin project.
+version: 0.0.1
+author:
+homepage:
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+# For information on the generic Dart part of this file, see the
+# following page: https://dart.dev/tools/pub/pubspec
+
+# The following section is specific to Flutter.
+flutter:
+  # This section identifies this Flutter project as a plugin project.
+  # The androidPackage and pluginClass identifiers should not ordinarily
+  # be modified. They are used by the tooling to maintain consistency when
+  # adding or updating assets for this project.
+  plugin:
+    androidPackage: com.example.flutter_android_lifecycle
+    pluginClass: FlutterAndroidLifecyclePlugin
```

Comment:
We do not support Android plugins that are not registered (this is the first use case for such a plugin), so this class must be an existing class that can be registered for applications to build.

Reply:
Done